src/components/CompareResults/CompareResultsTable.tsx

```diff
@@ -23,7 +23,11 @@ function CompareResultsTable(props: CompareResultsProps) {
       {!compareResults.loading && (
         <>
           <CompareTableStatus />
-          <TableContainer component={Paper}>
+          <TableContainer component={Paper} 
+            sx={{ 
+              marginBottom: '70px', 
```

Comment:
Do you think this table could benefit from any other improvements so that this PR could have more impact on the project's UI/UX?

Reply:
I believe it could. Please, give me a moment to illustrate some improvements I think it could benefit from in design.

Reply:
Hello @Carla-Moz 

I am sorry about the delay...
These are some improvements I am proposing.
Here is the link to the live version on  👉 [Figma](https://www.figma.com/proto/Gtkt1gnigDgT7pAfD4Q19B/Suggestions-for-PerfComare-Table?node-id=1-2&scaling=scale-down-width&page-id=0%3A1)
![proposed improvement1](https://user-images.githubusercontent.com/60399800/227307877-06841642-6e57-4755-9c48-4298f9172352.gif)